src/Twig/HomeBlockExtension.php

```diff
@@ -2,18 +2,23 @@
 
 namespace AppBundle\Twig;
 
+use AppBundle\Entity\Adherent;
 use AppBundle\Entity\HomeBlock;
 use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Twig\Environment;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFunction;
 
 class HomeBlockExtension extends AbstractExtension
 {
     private $manager;
+    private $tokenStorage;
 
-    public function __construct(ObjectManager $manager)
+    public function __construct(ObjectManager $manager, TokenStorageInterface $tokenStorage)
```

Comment:
useless

Reply:
.... totaly forgot to delete it after the revert :'(